portfolio.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,700;1,700&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <title>Homework 2</title>
+</head>
+
+<body>
+  <header>
+    <a href="index.html" class="logo">WEbStudio</a>
+
+    <nav class="nav">
+      <ul class="nav-list">
+        <li class="nav-item">
+          <a class="nav-link" href="index.html">Студія</a>
+        </li>
+        <li>
+          <a class="nav-link" href="portfolio.html">Портфоліо</a>
+        </li>
+        <li>
+          <a class="nav-link" href="#contacts">Контакти</a>
+        </li>
+      </ul>
+    </nav>
+
+    <address class="contact">
+      <a class="contact-link" href="mailto:info@devstudio.com">info@devstudio.com</a>
+      <a class="contact-link" href="tel:+380961111111">+38 096 111 11 11</a>
+    </address>
+  </header>
+  <main>
+    <section class="filters">
+      <ul>
+        <li>
+          <button type="button" class="filter-button">
+            Усі
+          </button>
+        </li>
+        <li>
+          <button type="button" class="filter-button">
+            Веб-сайти
+          </button>
+        </li>
+        <li>
+          <button type="button" class="filter-button">
+            Додатки
+          </button>
+        </li>
+        <li>
+          <button type="button" class="filter-button">
+            Дизайн
+          </button>
+        </li>
+        <li>
+          <button type="button" class="filter-button">
+            Маркетинг
+          </button>
+        </li>
+      </ul>
```

Comment:
add classes and wrap section content into a `div,container` tag

Reply:
Container is homework 3

Reply:
Li class="button-item" is good name of class?